services/ui-src/src/config.ts

```diff
@@ -23,11 +21,8 @@ const config = {
     REDIRECT_SIGNIN: window._env_.COGNITO_REDIRECT_SIGNIN,
     REDIRECT_SIGNOUT: window._env_.COGNITO_REDIRECT_SIGNOUT,
   },
-  STAGE: window._env_.STAGE,
   DEV_API_URL: window._env_.DEV_API_URL,
```

Comment:
Pretty sure DEV_API_URL should be destroyed everywhere.

Reply:
it can't be, then the unit tests fail.

Reply:
Totally true. I just know in my bones that there's no good reason for it in our current setup. We could rip it out of print and the tests about print and the world would be better for it.

That said, this isn't what the cdk conversion is about so we can leave it if you don't want to do it.

Reply:
Chatted with @bangbay-bluetiger .  We'll remove DEV_API_URL outside of the CDK migration.